app/src/main/java/com/breadwallet/BreadApp.java

```diff
@@ -55,16 +46,23 @@ public void onCreate() {
         AnalyticsManager.init(this);
         AnalyticsManager.logCustomEvent(BRConstants._20191105_AL);
 
-        if(BuildConfig.DEBUG) Timber.plant(new Timber.DebugTree());
+        if (BuildConfig.DEBUG) Timber.plant(new Timber.DebugTree());
```

Comment:
Was this the linter or Andhika manual @andhikayuana ?

Reply:
I do manually before commit

![Screenshot 2024-12-13 at 17 05 18](https://github.com/user-attachments/assets/b555e3ae-90fb-4863-9533-a23aefbb6748)
![Screenshot 2024-12-13 at 17 05 35](https://github.com/user-attachments/assets/c0a39f30-7cbb-4c1d-8b47-da5766e0f7f1)